integration/test/basic/main_test.go

```diff
@@ -164,17 +55,25 @@ func init() {
 func TestMain(m *testing.M) {
 	ctx := context.Background()
 
+	var err error
+
 	{
-		c := e2esetup.Config{
-			HelmClient: helmClient,
-			Setup:      k8sSetup,
+		apps := []apptest.App{
+			{
+				CatalogName:   catalogName,
+				Name:          appName,
+				Namespace:     metav1.NamespaceSystem,
+				SHA:           env.CircleSHA(),
+				ValuesYAML:    "e2e: true",
```

Comment:
This is used to disable auto discovery since it can't work for KIND clusters.

https://github.com/giantswarm/cluster-autoscaler-app/blob/daa81b226c133b96a8d462e6f60a77408097ec3d/helm/cluster-autoscaler-app/templates/deployment.yaml#L44-L45

Reply:
That looks like quite some black magic right there. 😅

Reply:
True but the problem is we need it to get the app installed in a KIND cluster. I've added a comment to explain.